### PR TITLE
formats release notes for 19.4.1 for better readability

### PR DESCRIPTION
**Pull request by dadjeibaah**
Signed-off-by: Dennis Adjei-Baah <dennis@buoyant.io>

### CODE REVIEW

**Comment by scottcarol** [comment]
⭐️

File: CHANGES.md [hunk 1]
```diff
@@ -240,19 +240,20 @@ UUID from the `linkerd-config` ConfigMap!
   * The `stat` command now always shows the number of open TCP connections
   * **Breaking Change** Removed the `--disable-external-profiles` flag from the
     `install` command; external profiles are now disabled by default and can be
-    enabled with the new `--enable-external-profiles` flag
+     enabled with the new `--enable-external-profiles` flag
```
Comment (siggy) [inline CHANGES.md]: ws ;)

File: CHANGES.md [hunk 2]
```diff
@@ -240,19 +240,20 @@ UUID from the `linkerd-config` ConfigMap!
...
 * Controller
   * The auto-inject admission controller webhook is updated to watch pods
-    creation and update events; with this change, proxy auto-injection now works
-    for all kinds of workloads, including StatefulSets, DaemonSets, Jobs, etc
+    creation and update events; with this change, proxy auto-injection now
```
Comment (siggy) [inline CHANGES.md]: i think this was already wrapped to 80?

File: CHANGES.md [hunk 3]
```diff
@@ -240,19 +240,20 @@ UUID from the `linkerd-config` ConfigMap!
...
+    Jobs, etc
 * Proxy
   * Some `l5d-*` informational headers have been temporarily removed from
     requests and responses because they could leak information to external
     clients
 * Web UI
   * The topology graph now shows TCP stats if no HTTP stats are available
-  * Improved table display on the resource detail page for resources with
-    TCP-only traffic
+  * The resource detail page no longer shows blank tables if the resource only
```
Comment (siggy) [inline CHANGES.md]: intentional rewrite? if so, consider rewording same line-item in the 2.3.0 section

Comment (dadjeibaah) [reply CHANGES.md]: Hmm, I didn't rewrite this prior to the merge conflict, thanks for catching that.

Comment (dadjeibaah) [reply CHANGES.md]: Must have been during the merge conflict

File: CHANGES.md (hunk 2 above)
Comment (dadjeibaah) [reply CHANGES.md]: I think the the word `works` was just over 80 chars and so I moved that down to the next line which caused the other lines to be formatted.

**Comment by dadjeibaah** [comment]
Looks like this branches changes were already made in the last release so going to close this in favor of the current change.

**Pull request closed without merging**